odoo/addons/base/models/res_lang.py

```diff
@@ -233,52 +266,42 @@ def _get_active(self, code):
             return 'en_US'
         return code
 
-    @tools.ormcache('self.code', 'monetary')
```

Comment:
The ormcache is useless, becasue when `self.code` is checked, `self.grouping`,  `self.thousands_sep`, `self.decimal_point` usually will be prefetched